### PR TITLE
894 exclude reviewer

### PR DESCRIPTION
**Pull request by norisalsaadie**
Closes https://github.com/libero/reviewer/issues/894

### CODE REVIEW

File: src/initial-submission/components/EditorsForm.tsx [hunk 1]
```diff
@@ -161,6 +177,37 @@ const EditorsForm = ({ initialValues, ButtonComponent }: StepProps): JSX.Element
...
+                />
+                <MultilineTextField
+                    id="opposedReviewersReason"
+                    register={register}
+                    labelText={t('editors.opposed-reviewering-editor-reason-label')}
+                    invalid={errors && errors.opposedReviewersReason !== undefined}
+                    helperText={errors && errors.opposedReviewersReason ? errors.opposedReviewersReason.message : null}
+                />
+            </ExcludedToggle>
+            {/*TODO: translationforprefix*/}
```
Comment (diversemix) [inline src/initial-submission/components/EditorsForm.tsx:210]: Can we put in this translation?

Comment (norisalsaadie) [reply src/initial-submission/components/EditorsForm.tsx:210]: Which part?

Comment (norisalsaadie) [reply src/initial-submission/components/EditorsForm.tsx:210]: I believe it's already translated. Th is is a left over todo from the merge. I'll remove it

File: src/initial-submission/components/EditorsForm.tsx [hunk 2]
```diff
@@ -56,6 +57,11 @@ const EditorsForm = ({ initialValues, ButtonComponent }: StepProps): JSX.Element
                 [['name', 'email']],
             ),
         ),
+        opposedReviewingEditors: yup.array().max(2, t('opposed-reviewering-editors-max')),
+        opposedReviewersReason: yup.string().when('opposedReviewingEditors', {
+            is: editors => !!editors.length,
+            then: yup.string().required(t('editors.validation.opposed-reviewering-editor-reason-required')),
```
Comment (Cooryd) [inline src/initial-submission/components/EditorsForm.tsx:63]: Can we get a unit test to check this validation message is shown if its left blank after selecting an editor?

Comment (Cooryd) [reply src/initial-submission/components/EditorsForm.tsx:63]: see https://github.com/libero/reviewer-client/blob/511e475ba39ec709e43b37b8bc4bd04fa419163e/src/initial-submission/components/EditorsForm.test.tsx#L451 for how we've done that for other inputs on page

Comment (norisalsaadie) [reply src/initial-submission/components/EditorsForm.tsx:63]: Makes sense. Was going to do that as part of the end to end test.

Comment (Cooryd) [reply src/initial-submission/components/EditorsForm.tsx:63]: The test bellow that is probably a better example (as that ones a test to ensure no validation is shown). Using the initialValues to load in with opposed Reviewers but have no reason value will be your best bet

**Pull request merged**